content/_index.html

```diff
@@ -5,89 +5,176 @@
-
-	<div class="lead">
-		<div class="image left">
-			<object type="image/svg+xml" data="serving.svg">
-			</object>
-		</div>
-		<div class="text">
-			<h4>Model serving</h4>
-			<p>A TensorFlow Serving container to export trained TensorFlow models to Kubernetes. We also integrate with Seldon Core, an open source platform for deploying machine learning models on Kubernetes, and NVIDIA TensorRT Inference Server for maximized GPU utilization when deploying ML/DL models at scale.</p>
-		</div>
-	</div>
-
-	<div class="lead">
-		<div class="image right">
-			<object type="image/svg+xml" data="pipelines.svg">
-			</object>
-		</div>
-		<div class="text">
-			<h4>Pipelines</h4>
-			<p>A solution to deploy and manage end-to-end machine learning workflows, providing rapid and reliable ML experimentation.</p>
-		</div>
-	</div>
-
-	<div class="lead">
-		<div class="image left">
-			<object type="image/svg+xml" data="multi-framework.svg" alt="Multi-framework">
-			</object>
-		</div>
-		<div class="text">
-			<h4>Multi-framework</h4>
-			<p>Our development plans go beyond TensorFlow, and we are working hard to include PyTorch, MXNet, Chainer, and more. We also integrate with Ambassador for ingress, Nuclio as a fast multi-purpose serverless framework, and Pachyderm for managing your data science pipelines.</p>
-		</div>
-	</div>
-
-	<div class="lead">
-		<div class="image right">
-			<img src="icn-people.svg" alt="People Icon">
-		</div>
-		<div class="text">
-			<h4>Community</h4>
-			<p>We are an open and welcoming community of software developers, data scientists, and organizations that are working to make it easier to develop and deploy scalable ML workflows across the industry. <a href="/docs/about/contributing/">Start contributing.</a>
-		</div>
-	</div>
+  <div class="container">
+    <div class="card-deck">
+      <div class="card border-primary">
+        <img
+          src="logos/Jupyter.png"
+		  class="card-img-top"
+		  draggable="false"
+          style="padding: 2rem;"
+          alt="Jupyter Logo"
+        />
+        <div class="card-body bg-primary">
+          <h5 class="card-title text-light section-head">Notebooks</h5>
+          <p class="card-text text-light">
+            Kubeflow includes services to create and manage interactive <a target='_blank' class='text-light' href="https://jupyter.org/">Jupyter
+            notebooks</a>. You can choose custom images, and allocate specific
+            compute resources based on your data science needs. This
+            comprehensive experience allows you to create workflows locally, and
+            deploy them with ease.
+          </p>
+        </div>
+      </div>
+      <div class="card border-primary">
+        <img
+          src="logos/TensorFlow.png"
+		  class="card-img-top"
+		  draggable="false"
+          style="padding: 2rem;"
+          alt="TensorFlow"
+        />
+        <div class="card-body bg-primary">
+          <h5 class="card-title text-light section-head">
```

Comment:
@abhi-g we currently give TF special treatment, in that it has a card all to itself. Are the following now all first-class citizens of Kubeflow: Chainer, MPI, MXNext, PyTorch?

If some or all are on equal footing with TF, then we have some options:

* Convert the TF card to a **Model training** card, and list all the supported systems - much as we do in the **Model serving** card.
* Keep the TF card as it is, and add the other serving components in a card called something like **More training systems**.
* Other...

<I haven't reviewed the text in this section yet. I'll wait until we hear about the other training systems.>

Reply:
Here's the reply from abhi-g, copied from his single long comment:

>MXNet and Chainer have much lower activity as compared to TF and PyTorch in
the respective repos. I consider TensorFlow and TFX to still be the primary
supported frameworks. We have put extensive effort around TF and TFX
components to ensure they run well and enable multiple areas of the ML
workflow eg. Transforms, data validation, model evaluation and validation.
There is significant effort by TFX team as well to ensure great experience
with KF pipelines.